data/distro.yml

```diff
@@ -477,3 +477,21 @@
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
       </li>
     </ol>
+
+- name: "PureOS"
+  logo: "pureos.svg"
+  info:  >
+    <ol class="distrotut">
+      <li>
+        <h2>Add the Flathub repository</h2>
+        <p>Flatpak is installed by default on PureOS. You just need to enable the Flathub repository. There are two ways.</p>
```

Comment:
I think that instead of providing "two ways to enable Flathub", it would be less confusing to use the same text that is used, for example, by [Fedora instructions](https://flatpak.org/setup/Fedora/).

Reply:
Used the Fedora Instructions instead